community/sql.gitattributes

```diff
@@ -1,4 +1,30 @@
-# Basic .gitattributes for sql files
+# Basic .gitattributes for SQL files
 
 *.sql linguist-detectable=true
 *.sql linguist-language=sql
+
+# Handle line endings automatically for files detected as text
+# and treat them as LF on check-in and CRLF when checked out.
+* text=auto eol=crlf
```

Comment:
This would force `crlf` for even non-Windows users, so I don't think this line should have it.

Regarding, sql-specific files, should those have `crlf` or should this be chosen by the user?